_includes/code/embedded.instantiate.simple.mdx

```diff
@@ -2,7 +2,18 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 <Tabs groupId="languages">
-  <TabItem value="py" label="Python">
+  <TabItem value="py4" label="Python (v4)">
```

Comment:
For the new snippets, I think it would be preferable to create a new .py file and import them with the custom snippet component.